servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ErrorHandlingTest.java

```diff
@@ -263,9 +263,12 @@ private void setUp(TestMode testMode, GrpcExecutionStrategy serverStrategy,
         this.requestPublisher = requestPublisher;
         serverContext = GrpcServers.forAddress(localAddress(0)).appendHttpServiceFilter(serviceFilterFactory)
                 .executionStrategy(serverStrategy).listenAndAwait(serviceFactory);
+        final StreamingHttpClientFilterFactory pickedClientFilterFactory = clientFilterFactory;
         GrpcClientBuilder<HostAndPort, InetSocketAddress> clientBuilder =
-                GrpcClients.forAddress(serverHostAndPort(serverContext)).appendHttpClientFilter(clientFilterFactory)
-                .executionStrategy(clientStrategy);
+                GrpcClients.forAddress(serverHostAndPort(serverContext))
+                        .initializeHttp(builder -> builder
+                                .appendClientFilter(pickedClientFilterFactory)
+                                .executionStrategy(clientStrategy));
```

Comment:
If we migrate to `SingleAddressHttpClientBuilder#executionStrategy`, we should also migrate to `HttpExecutionStrategy` instead of `GrpcExecutionStrategy`.